### DISCUSSION TITLE
Create a private Github pages

### DISCUSSION THREAD

**Discussion by sfullerbeckman** [community/community#23362] [Other Feature Feedback, Questions, & Ideas]
<p>I have a Github organization, and I am going to create a private or internal repo in that organization. That repo is going to host a Github Pages site. But I want the site to be private and not publically accessible by people outside the organization. How do I do this?</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>We support what you are describing and call it Pages access control. This is available only to Enterprise customers today and documented here: <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site" class="inline-onebox">Changing the visibility of your GitHub Pages site - GitHub Docs</a>.</p>
<p>Hope it helps,</p>

**Comment by malaniarpit** [comment]
<p>Hi Yoann,</p>
<p>We are not able to see the page visibility option with the new section introduced for GH Pages.</p>
<p><img src="https://user-images.githubusercontent.com/4762437/181076540-e52a3cf7-4eef-4c31-9694-08c4e862dc6a.png" alt="public-or-private-visibility" data-base62-sha1="ib7b2gXKp2K0dPDCyrXQvLM3cRp" width="502" height="293"></p>
<p>-Arpit</p>

**Comment by sfullerbeckman** [comment]
<p>Yes, thank you. That helps.</p>

**Comment by ernest-phillips** [comment]
<p>Hi Arpit, the Access Control feature is only available to GitHub Pages sites is available in private repositories with GitHub Enterprise Cloud.</p>

**Comment by 0xdevalias** [comment]
I know that access control for GitHub pages is available for GitHub Enterprise Cloud as per:

- https://github.blog/changelog/2021-01-21-access-control-for-github-pages/
- https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site

But is there any way to get access to that feature as a normal 'private' `.com` GitHub Organisation? I would love to be able to keep everything within GitHub rather than needing to look at external providers, but right now we're sort of left with no other choice unfortunately?

And if there is no way that it's possible currently, I would love it could be raised as a feature request.
